mycroft/stt/__init__.py

```diff
@@ -22,6 +22,12 @@
 from mycroft.configuration import ConfigurationManager
 from mycroft.util.log import getLogger
 
+
+#imports for json handling, requests and string formatting for KaldiSTT
```

Comment:
I've implemented all changes and it's working fine. But I don't know how to rebase with dev and doesn't have the time to find out now. Here is my work from now: 

`
class KaldiSTT(STT):
    def __init__(self):
        super(KaldiSTT, self).__init__()

    def execute(self, audio, language=None):
        language = language or self.lang
          
        config = ConfigurationManager.get().get("stt", {})
       
        response = requests.post(config.get("uri"), data=audio.get_wav_data())        
        return get_response(response)

def get_response(response):
    try:
      hypotheses = response.json()["hypotheses"]
      return re.sub(r'\s*\[noise\]\s*', '', hypotheses[0]["utterance"])
    except:
      return None
        `